test/driver.js

```diff
@@ -468,12 +472,13 @@ var Driver = (function DriverClosure() {
               var textLayerContext = textLayerCanvas.getContext('2d');
               textLayerContext.clearRect(0, 0,
                 textLayerCanvas.width, textLayerCanvas.height);
+              var enhanceText = task.id.includes('enhance') ? true : false;
```

Comment:
Wouldn't just `enhance` be enough though, since the type is already set to `text`?